generators/server/templates/package.json.ejs

```diff
@@ -33,6 +33,6 @@
     "@openapitools/openapi-generator-cli": "1.0.12-4.3.0"
   },
   "engines": {
-    "node": ">=8.9.0"
+    "node": "12.16.1"
```

Comment:
Do you mean to force a specific version?

Reply:
@machima: Not really, that was a mistake. Thanks. :+1: